lms/templates/student_profile/learner_profile.underscore

```diff
@@ -0,0 +1,28 @@
+<div class="profile <%- ownProfile ? 'profile-self' : 'profile-other' %>">
+    <div class="wrapper-profile-field-account-privacy"></div>
+
+    <div class="section wrapper-profile-sections account-settings-container">
+        <div class="wrapper-profile-section-one">
+            <div class="profile-photo">
+                <img src="<%- profilePhoto %>">
+            </div>
+
+            <div class="profile-section-one-fields">
+            </div>
+        </div>
+        <div class="ui-loading-error is-hidden">
+            <i class="fa fa-exclamation-triangle message-error" aria-hidden=true></i>
+            <span class="copy"><%- gettext("An error occurred. Please reload the page.") %></span>
+        </div>
+        <div class="section wrapper-profile-section-two">
+            <div class="profile-section-two-fields">
+                <% if (!showFullProfile) { %>
+                    <label class="profile-private--message">
+                        <%- gettext('This edX learner is not currently sharing their profile details.') %>
```

Comment:
Text: This edX learner is currently sharing a limited profile.

Comment: Should this message be different for the user herself and for other page viewers? It's a little strange for me to see "This edX learner is not currently sharing their profile details" on my own profile page. If we don't want to have different messages, can we indicate somewhere on the UI that other users will see the profile page exactly as it looks here - maybe a "Other edX learners will see your profile exactly as it appears below" under the "edX learners can see my" line? (Maybe this is a follow-on story?)

Another potential follow-on story: Let users know that if they make changes to the full profile, those changes won't be deleted if the user switches to the limited profile and then back to the full profile. It could be a message like "This information will be saved but not visible when you switch to a limited profile" (though said much more elegantly than this) above the "About Me" text box. (I would add this above the text box rather than as placeholder text so that it's always visible.)